tools/decompress_baserom.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: © 2024 ZeldaRET
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+import hashlib
+import io
+import struct
+from pathlib import Path
+import argparse
+
+import crunch64
+import ipl3checksum
+import zlib
+
+def decompressZlib(data: bytes) -> bytes:
+    decomp = zlib.decompressobj(-zlib.MAX_WBITS)
+    output = bytearray()
+    output.extend(decomp.decompress(data))
+    while decomp.unconsumed_tail:
+        output.extend(decomp.decompress(decomp.unconsumed_tail))
+    output.extend(decomp.flush())
+    return bytes(output)
+
+def decompress(data: bytes, is_zlib_compressed: bool) -> bytes:
+    if is_zlib_compressed:
+        return decompressZlib(data)
+    return crunch64.yaz0.decompress(bytes(data))
+
+FILE_TABLE_OFFSET = {
```

Comment:
Could we wait to add other versions for now? For example I'm not yet sure how to name NTSC-J and NTSC-U.

Reply:
You would prefer this list to only have `gc-eu-mq-dbg` and `gc-eu-mq` then?

Reply:
Yeah (well technically only `gc-eu-mq-dbg` for the current state of the repo, but `gc-eu-mq` will be added soon anyway)